recipe/meta.yaml

```diff
@@ -26,51 +26,47 @@ requirements:
   build:
     # pythran code needs clang-cl on windows
     - clang                 # [win]
+    - lld     # [win]
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
     - llvm-openmp >=4.0.1   # [osx]
     - m2-patch              # [win]
     - patch                 # [not win]
   host:
     - python
-    - cython >=0.29.24,<3.0
+    - cython >=0.29.24
     - llvm-openmp >=4.0.01  # [osx]
```

Comment:
I would prefer hard pinning in `host`: `llvm-openmp 14.0.6` to prevent issues if there will be released an incompatible version of llvm-openmp

Reply:
Added!